introduction.md

```diff
@@ -3,5 +3,10 @@
 - wealth (health?)
 - taste
 
+##Deadly skills
+* Git
+* JavaScript
+* Python
+
 
 edit: Electives is my favorite code school path
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?